pkg/apis/operator/v1alpha1/glustercluster_types.go

```diff
@@ -1,22 +1,60 @@
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
+	resource "k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// GlusterStorageTarget defines a storage target
+type GlusterStorageTarget struct {
+	Name    string       `json:"name,omitempty"`
+	Address []string     `json:"address"`
```

Comment:
May be Addresses

Reply:
I feel the yml in document @JohnStrunk had created it was named `Address` so kept like that.
@JohnStrunk ^^^ ?

Reply:
Changed it to `Addresses`

Reply:
Good catch. @humblec is correct. Since it's a list, convention would indicate a plural name.